### PR TITLE
Solved App crashes when entering an existing email ID saved in `info_table`

### LINKED ISSUE #116: Bug: App crashes when entering an existing email ID saved in `info_table`
## Description  
The application crashes when the user inputs an email ID that already exists in the `info_table` during form submission. This disrupts the user flow and prevents data from being processed correctly.

## Steps to Reproduce  
1. Open the app and navigate to the form that collects user details.  
2. Enter an email ID that is already stored in the `info_table`.  
3. Click the **Submit** button to save the form data.  

## Expected Behavior  
The app should display a user-friendly message (e.g., "Email ID already exists. Please use another email.") and prevent duplicate entries without crashing.  

## Actual Behavior  
The app crashes immediately upon detecting the duplicate email ID.  

## Possible Root Cause  
- Unhandled exception when checking for duplicate email IDs in the `info_table`.  
- Missing error handling logic during form submission for email validation.  

## Suggested Fix  
- Add a validation check for existing email IDs in `info_table` before saving.  
- Handle exceptions gracefully and display a meaningful error message if the email already exists.  

## Additional Information  
- 
image

@Harshdev098 pls assign me this issue and give me level3,gssoc extended and hacktober fest tags

### PR DESCRIPTION
**Pull request by ygowthamr**
#### Overview
This pull request addresses the issue of duplicate email registrations in the application. It implements a check to verify whether an email address already exists in the database before allowing a user to register. If a duplicate email is detected, the application will now display an appropriate message to the user, preventing the registration process and enhancing the overall user experience.

#### Changes Made
- **Email Existence Check**: Added a query to check if the email already exists in the `info_table` during the registration process.
- **User Feedback**: Implemented a user-friendly error message when a duplicate email is found, guiding the user to either choose a different email or log in to their existing account.
- **Code Refactoring**: Ensured proper error handling and connection management during the database operations.
- **Code Changes**: 
- I have added/modified code in form_filling.html file in the following lines : 84-86,115,141-143.
- In form_db.js file the changes are , modified info function

#### Example Error Message
- "This email address is already registered. Please use a different email."

#### Additional Notes
- 
image

Fixes #116
@Harshdev098 Pls add labels like label3,gssoc, and hacktoberfest accepted.

### CODE REVIEW

**Comment by ygowthamr** [comment]
@Harshdev098 
I have resolved all the conflicts with the code pls merge my pr.

File: public/form_filling.html [hunk 1]
```diff
@@ -138,6 +148,8 @@
                 const course = document.getElementById('course').value.toUpperCase();
                 const year = document.getElementById('year').value;
                 const result = document.getElementById('result');
+                const result2 = document.getElementById('result2');
```
Comment (Harshdev098) [inline public/form_filling.html:151]: the result2 variable is replicated

File: public/form_filling.html [hunk 2]
```diff
@@ -84,7 +91,7 @@
                     <option value="Lakshadweep">Lakshadweep</option>
                     <option value="Ladakh">Ladakh</option>
                     <option value="Puducherry">Puducherry</option>
-                    <option value="Jammu and Kashmir">Jammu and Kashmir</option>
+                    <option value="Jammu and Kashmir">Jammu and Kashmir</option
```
Comment (Harshdev098) [inline public/form_filling.html:94]: Don't delete the tag

File: public/form_filling.html [hunk 3]
```diff
@@ -48,6 +48,13 @@
             <div class="form-group">
                 <label>State</label>
                 <select name="state" id="state">
+                    <option value="Uttar Pradesh">Uttar Pradesh</option>
+                    <option value="Madhya Pradesh">Madhya Pradesh</option>
+                    <option value="Tamil Nadu">Tamil Nadu</option>
+                    <option value="Maharashtra">Maharashtra</option>
```
Comment (Harshdev098) [inline public/form_filling.html:54]: Also you have duplicated the options in State like UP and MP...

File: file_upload/form_db.js [hunk 4]
```diff
@@ -1,62 +1,96 @@
...
+        connection.release();
+        return res.sendStatus(400);
+      }
+
+      // Proceed to insert the data if the email does not exist
+      const sql = "INSERT INTO info_table VALUES (?,?,?,?,?,?,?)";
+      const sqlInsert = mysql.format(sql, [
+        userid,
+        name,
+        email,
```
Comment (Harshdev098) [inline file_upload/form_db.js:50]: Also make these mysql fields in one line

**Review by Harshdev098 — CHANGES_REQUESTED** [review]

**Comment by Harshdev098** [comment]
Also update your PR description to add the issue number you are solving `Fixes #<issue_number>`

**Comment by ygowthamr** [comment]
@Harshdev098 
Updated pls check and let me know

File: public/form_filling.html [hunk 5]
```diff
@@ -138,6 +140,8 @@
                 const course = document.getElementById('course').value.toUpperCase();
                 const year = document.getElementById('year').value;
                 const result = document.getElementById('result');
+                const result2 = document.getElementById('result2');
+                const result3 = document.getElementById('result3')
```
Comment (Harshdev098) [inline public/form_filling.html:144]: The result2 variable is declare twice in this html code, its giving error

**Review by Harshdev098 — CHANGES_REQUESTED** [review]

**Comment by ygowthamr** [comment]
@Harshdev098 
There is some issue regarding the docker configuration from you side pls check

**Comment by Harshdev098** [comment]
Will see it

**Comment by ygowthamr** [comment]
@Harshdev098 
Now i think the issue from your side is solved pls review the code now

**Comment by ygowthamr** [comment]
@Harshdev098 
I think there is some issue in pushing the code i will re pull this thing so that there will be not such errors 

Pls consider other prs

**Comment by Harshdev098** [comment]
Yah, its resolved

**Comment by ygowthamr** [comment]
@Harshdev098 
Thank you

**Pull request merged**